cli/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "ethabi-cli"
-version = "12.0.0"
```

Comment:
I'd prefer if we didn't touch the ethabi version at all. IMHO we should only make the changes that we absolutely require and otherwise base our version on top of upstream, using the same version number as theirs to keep the delta minimal.

Reply:
This applies to a couple other files as well.